src/cpp/session/modules/NotebookData.R

```diff
@@ -230,6 +230,16 @@
     paste0(" [", dim_desc(x), "]" )
   }
 
+  needs_obj_sum <- function(x) {
+    if (!is.list(x)) return(FALSE)
+    if (!is(x, "vctrs_vctr")) return(TRUE)
```

Comment:
```suggestion
    if (!inherits(x, "vctrs_vctr")) return(TRUE)
```

Since `is()` is normally used with S4 objects.